tests/compilation/test_predictor_rl.py

```diff
@@ -24,13 +24,13 @@ def test_predictor_env_reset_from_string() -> None:
 
 @pytest.mark.parametrize(
     "figure_of_merit",
-    ["expected_fidelity", "critical_depth"],
+    ["expected_fidelity", "critical_depth", "expected_success_probability"],
 )
 def test_qcompile_with_newly_trained_models(figure_of_merit: reward.figure_of_merit) -> None:
     """Test the qcompile function with a newly trained model."""
     """ Important: Those trained models are used in later tests and must not be deleted. """
 
-    device = "ibm_montreal"
+    device = "ionq_harmony"
```

Comment:
What was the reason again to change this device?

Reply:
Could you add this information as a comment to avoid that we in the future change the device and will be surprised by unexpected behavior in other tests?